### PR TITLE
Fix update changes

### PR DESCRIPTION
**Pull request by CarstenKoenig**
this one should Fix https://github.com/fsharp-editing/Forge/issues/68 that happens after 

    paket update

as said in the issue without the update Forge will not build at all under Linux - this one is working again

#### remark:
it seems as if the `-h` option is not working with this version of Argu anymore and you have to fully qualify the `--help`

### CODE REVIEW

File: src/Forge/Forge.fsproj [hunk 1]
```diff
@@ -60,6 +60,7 @@
     <Content Include="Tools/Paket/paket.bootstrapper.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Compile Include="obj/Release/.NETFramework,Version=v4.5.AssemblyAttribute.fs" />
```
Comment (Krzysztof-Cieslak) [inline src/Forge/Forge.fsproj]: Looks like this thing break build.

Comment (CarstenKoenig) [inline src/Forge/Forge.fsproj]: hmm ... nasty ... that was included by the recommended `paket update` - my local build on linux had all tests pass

recomendations?

Comment (Krzysztof-Cieslak) [inline src/Forge/Forge.fsproj]: Let's just try to remove this line. No idea why Paket added it.

Comment (CarstenKoenig) [inline src/Forge/Forge.fsproj]: ok - gonna try as soon as I'm home again later

**Comment by CarstenKoenig** [comment]
ok ... now AppVeyor has some strange LoaderProblem in the *RunTests* phase ... travis and my local system are both happy ... any comment/clue how I could fix this?

Is this acceptable?

**Comment by Krzysztof-Cieslak** [comment]
I just need to port test suite to Expecto.... NUnit causes too much random problems.

**Comment by Krzysztof-Cieslak** [comment]
Thanks a lot for the fix!

**Pull request merged**